pkg/pgclient/config.go

```diff
@@ -82,6 +83,7 @@ func ParseFlags(fs *flag.FlagSet, cfg *Config) *Config {
 	fs.BoolVar(&cfg.EnableStatementsCache, "db.statements-cache", defaultDbStatementsCache, "Whether database connection pool should use cached prepared statements. "+
 		"Disable if using PgBouncer")
 	fs.BoolVar(&cfg.AsyncAcks, "metrics.async-acks", false, "Acknowledge asynchronous inserts. If this is true, the inserter will not wait after insertion of metric data in the database. This increases throughput at the cost of a small chance of data loss.")
+	fs.IntVar(&cfg.InvertedLabelsCacheSize, "metrics.inverted-lables-cache-size", 500000, "Maximum number of labelIDs to cache. Inverted lables cache reduces number of DB calls when ingesting new series.")
```

Comment:
```suggestion
	fs.IntVar(&cfg.InvertedLabelsCacheSize, "metrics.cache.label-ids.size", 500000, "Maximum number of label-ids to cache. This reduces number of DB calls when ingesting new series.")
```
We should abstract the invert indexing details from the naming. Plus, going with the convention of other cache nomenclature, it should follow.

The details regarding inverted index can be added as code comments for others to follow.

Reply:
Inverted is intentionally there because we already have labels cache ( `id` -> `label`).